src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/AddInstruction.cs

```diff
@@ -1,23 +1,25 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+// Do not edit this file manually. This file is auto-generated by AddInstruction.tt.
+
 using System.Diagnostics;
 using System.Dynamic.Utils;
 
 namespace System.Linq.Expressions.Interpreter
 {
     internal abstract class AddInstruction : Instruction
     {
-        private static Instruction s_int16, s_int32, s_int64, s_UInt16, s_UInt32, s_UInt64, s_single, s_double;
+        private static Instruction s_Int16, s_Int32, s_Int64, s_UInt16, s_UInt32, s_UInt64, s_Single, s_Double;
 
         public override int ConsumedStack => 2;
         public override int ProducedStack => 1;
         public override string InstructionName => "Add";
 
         private AddInstruction() { }
 
-        private sealed class AddInt32 : AddInstruction
+        private sealed class AddInt16 : AddInstruction
```

Comment:
I was thinking to push a separate PR before this one (but after https://github.com/dotnet/corefx/pull/14180) to sort all inner types in the order of the `s_` fields above, so the diff becomes significantly less here.